waku/waku_rln_relay/constants.nim

```diff
@@ -29,6 +32,14 @@ const
 const
   DefaultRlnTreePath* = "rln_tree.db"
 
+when defined(rln_v2):
+  const
+    # pre-processed "rln/waku-rln-relay/v2.0.0" to array[32, byte]
+    DefaultRlnIdentifier*: RlnIdentifier = [114, 108, 110, 47, 119, 97, 107, 117, 
```

Comment:
imo this is the best way to have the DefaultRlnIdentifier stored - if we do decide to have it as a command line argument for rln-relay we can process it into an array of 32 bytes at that point.